cli/azd/test/functional/cli_test.go

```diff
@@ -68,17 +69,28 @@ func Test_CLI_Version_PrintsVersion(t *testing.T) {
 	defer cancel()
 
 	cli := azdcli.NewCLI(t)
-	out, err := cli.RunCommand(ctx, "version")
+	textOutput, err := cli.RunCommand(ctx, "version")
 	require.NoError(t, err)
 
-	rn := os.Getenv("GITHUB_RUN_NUMBER")
-	if rn != "" {
-		version := os.Getenv("CLI_VERSION")
-		require.NotEmpty(t, version)
-		require.Contains(t, out, version)
-	} else {
-		require.Contains(t, out, fmt.Sprintf("azd version %s", internal.Version))
+	jsonOutput, err := cli.RunCommand(ctx, "version", "--output", "json")
```

Comment:
nit. I would usually prefer to create a new test than updating a current one.  A test update can be understood as if there was a test issue here. But we are just increasing the test cases here.
But it's fine :).  Thank you for adding this

Reply:
I agree with you if it was a test that was written to catch a specific regression.. But if it's a test that is performing some sort of validation, it can be extended if it makes sense to update. I ended up refactoring the code for readability